pkg/ioutil/ioutil.go

```diff
@@ -212,6 +212,7 @@ func CopyAligned(w *os.File, r io.Reader, alignedBuf []byte, totalSize int64) (i
 			if err = disk.DisableDirectIO(w); err != nil {
 				return remainingWritten, err
 			}
+			defer w.Sync() // sync only on unaligned writes
```

Comment:
w.Sync() should be called after aligned writes too in order to commit the metadata:
```
The O_DIRECT flag on its own makes an effort to transfer data synchronously, but does not give the guarantees of the O_SYNC
              flag that data and necessary metadata are transferred.
```